package.json

```diff
@@ -118,5 +118,19 @@
     "help": "yamscripts help",
     "yamscripts": "yamscripts compile",
     "__": "# NOTE: THESE SCRIPTS ARE COMPILED!!! EDIT yamscripts.yml instead!!!"
-  }
+  },
+  "preferences": [
+    {
+      "name": "performanceLog",
```

Comment:
I would like to keep the old name as it allows for 
```
simplePrefs.prefs.performanceLog
```
As oppose to
```
simplePrefs.prefs["performance.log"]
```
Not critical, can fix if reviewer insists